Cython/Compiler/TypeInference.py

```diff
@@ -228,10 +228,7 @@ def visit_ParallelStatNode(self, node):
             self.parallel_errors = True
 
         if node.is_prange:
-            child_attrs = node.child_attrs
-            node.child_attrs = ['body', 'target', 'args']
-            self.visitchildren(node)
-            node.child_attrs = child_attrs
+            self.visitchildren(node, attrs=['body', 'target', 'args'])
```

Comment:
I think you can use a constant tuple here to avoid the overhead of creating the list at call time.
```suggestion
            self.visitchildren(node, attrs=('body', 'target', 'args'))
```